toolkit/scss/_statistics.scss

```diff
@@ -0,0 +1,29 @@
+@import "_typography.scss";
+@import "_measurements.scss";
+
+%statistic {
+  @include core-14;
+
+  @include media(tablet) {
+    @include core-16;
+  }
+  @include media(tablet) {
+    padding-right: 10px;
+  }
+}
+
+.big-statistic {
+  @extend %statistic;
+  @include bold-36;
+}
+
+.statistic-name {
+  @extend %statistic;
+  @include bold-19;
+}
+
+.statistic-container {
+  @include inline-block;
+  padding-right: 7em;
+  padding-bottom: 3em;
+}
```

Comment:
This might be better to be grouped under the `statistics-container` class. This way the scope affected will be smaller.

```scss
.statistic-container {
  @include inline-block;
  padding-right: 7em;
  padding-bottom: 3em;

  .big-statistic {
    @extend %statistic;
    @include bold-36;
  }

  .statistic-name {
    @extend %statistic;
    @include bold-19;
  }
}
```